slacker/main.py

```diff
@@ -52,6 +55,19 @@ def check():
       sys.exit(-1)
   slacker_logger.info('All good: {} valid commands'.format(reg.count()))
 
+def build_prompt_completer():
+  """ Build REPL completion dictionary """
+  cmds = []
+  cmds_meta = {}
+  for cmd in Command.find_all():
+    cmd = cmd()
+    cmd_name = cmd.name()
```

Comment:
You could also pass the `Registrar` instance to this function so we don't have to re-search and make new instances of all commands a second time.